constraint-solver/src/symbolic_to_quadratic.rs

```diff
@@ -0,0 +1,38 @@
+use std::hash::Hash;
+
+use powdr_number::FieldElement;
+
+use crate::{
+    quadratic_symbolic_expression::QuadraticSymbolicExpression,
+    symbolic_expression::{BinaryOperator, SymbolicExpression, UnaryOperator},
+};
+
+pub fn symbolic_expression_to_quadratic_symbolic_expression<
```

Comment:
This essentially turn known variables into unknown variables. In essence, this turns an expression into a "only concretely known is known" QSE.